display_blocks.js

```diff
@@ -161,10 +157,116 @@ function parseObject(obj) {
     var nodeInstPVList = document.createElement("ul");
 
     nodeInstPVs.appendChild(nodeInstPVList);
+    nodeInstPVs.appendChild(nodeInstPVList);
     getDisplayRunInfo(nodeInstPVs, instrumentState.inst_pvs);
+
+    nodeInstTitle.appendChild(document.createTextNode(instrument));
+    nodeConfigTitle.appendChild(document.createTextNode("Configuration: " + instrumentState.config_name));
+
+    document.getElementById("config_name").appendChild(nodeConfigTitle);
 	
 	setVisibilityMode('block');
-}
+};
+
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function newPartOfTable(){
+	document.getElementById("next_part").removeAttribute("id");
+	document.getElementById("table_part").innerHTML = document.getElementById("table_part").innerHTML + "<th id = \"next_part\" style = \"padding: 10px; background-color:lightgrey ; border: black 2px solid\";></th>";
+};
+
+/**
+ * creates a Title at the top looking similar to the IBEX GUI
+ */
+function createTitle(obj){
```

Comment:
Parameter should be named something else. This function does not work on any arbitrary object `obj`